appeals/web/src/server/lib/validators/date-input.validator.js

```diff
@@ -102,6 +128,30 @@ export const createDateInputDateValidityValidator = (
 			)
 	);
 
+export const createDateInputEmptyFieldsValidator = (
```

Comment:
This seems identical to the `createDateInputNoEmptyFieldsValidator` function above, is that intentional?